src/wp-includes/rest-api/endpoints/class-wp-rest-application-passwords-controller.php

```diff
@@ -631,6 +632,8 @@ public function get_item_schema() {
 					'type'        => 'string',
 					'required'    => true,
 					'context'     => array( 'view', 'edit', 'embed' ),
+					'minLength'   => 3,
+					'pattern'     => '.*\S.*',
```

Comment:
[Per suggestions in the ticket](https://core.trac.wordpress.org/ticket/51941#comment:15), modifies the schema for a minimum length and exclusion of all whitespaces.

@TimothyBJacobs What do you think of the `minLength` of 3? Do you foresee a need of names of 1 or 2 characters?

Reply:
I think we might want to be conservative and stick with a `minLength` of `1`. This is doing an `mb_strlen` so in some cases I think a single character name may be appropriate in certain languages.

Reply:
Okay cool. Updated in commit https://github.com/WordPress/wordpress-develop/pull/898/commits/b257be908524ec0579a8f41305dbc423d65723f3